server/notifications.go

```diff
@@ -36,7 +72,20 @@ func SendNotification(tenantName string, title string, message string, msgType s
 		}
 	}
 	return nil
+}
 
+func saveSlackNotificationToQueue(s *Server, stream, tenantName string, notificationMsg *NotificationMsg) error {
+	subject := fmt.Sprintf("%s.%s", stream, tenantName)
```

Comment:
you can set the stream to wrap 1 subject as the stream name rather than <stream>.> and not send messages into different subjects, it will be more performant on the consumer side as well as here.